submissions/smillims/oop/app.js

```diff
@@ -0,0 +1,123 @@
+const block = {
+	width: 101,
+	height: 85
+}
+
+// Enemies our player must avoid
+const Enemy = function(x, y, player) {
+    // Variables applied to each of our instances go here,
+    // we've provided one for you to get started
+
+	 this.x = x;
+	 this.y = y;
+	 this.player = player;
+	 this.speed = 150 + (Math.random() * 100); //speed between 100 and 250;
+
+    // The image/sprite for our enemies, this uses
+	 // a helper we've provided to easily load images
+	 
+    this.sprite = 'images/enemy-bug.png';
+};
+
+// Update the enemy's position, required method for game
+// Parameter: dt, a time delta between ticks
+Enemy.prototype.update = function(dt) {
+    // You should multiply any movement by the dt parameter
+    // which will ensure the game runs at the same speed for
+	 // all computers.
+	 this.x += this.speed * dt;
+	 if (this.x > ctx.canvas.width + block.width){
+		 this.x = -101 * 2;
```

Comment:
`-101` resembles negated `block.width` or it is just a coincidence?

Check your entire code for mysterious numbers. They definitely have semantic meaning, thus worth defining as contants.

Reply:
Yeah, it's resembles negated `block.width`